gdp-src-build/conf/templates/dragonboard-410c.local.conf

```diff
@@ -0,0 +1,5 @@
+include templates/local.inc
+
+MACHINE = "dragonboard-410c"
+
+ACCEPT_EULA_dragonboard-410c = "1"
```

Comment:
@tom--pollard 

> ACCEPT_EULA_dragonboard-410c = "1"
I'll think about it. Please don't merge today. :)

Thanks for you comment.